addons/web/static/src/views/relational_model.js

```diff
@@ -1044,6 +1081,87 @@ export class Record extends DataPoint {
     }
 
+    _applyPropertyDefinitions(fieldName) {
+        const definitions = this.data[fieldName];
+        const propertiesField = this.fields[fieldName];
+
+        for (const definition of definitions) {
+            const propertyFieldName = `${fieldName}.${definition.name}`;
+            if (propertyFieldName in this.fields) {
+                continue;
+            }
+
+            let widget = definition.type;
+            if (["many2one", "many2many"].includes(definition.type)) {
+                if (!definition.comodel) {
+                    continue;
+                }
+
+                if (["res.users", "res.partner"].includes(definition.comodel)) {
+                    widget =
+                        definition.type === "many2one"
+                            ? "many2one_avatar"
+                            : "many2many_tags_avatar";
+                } else {
+                    widget = definition.type === "many2one" ? widget : "many2many_tags";
+                }
+            } else if (widget === "tags") {
+                widget = `property_tags`;
+            }
+
+            const parent = this.data[propertiesField.definition_record];
+
+            const relatedPropertyField = {
+                id: parent[0], //record.id,
+                fieldName,
+                displayName: parent[1], //record.display_name,
+            };
+
+            this.fields[propertyFieldName] = {
+                ...definition,
+                name: propertyFieldName,
+                sortable: false,
+                required: false,
+                relatedPropertyField,
+                propertyName: definition.name,
+                relation: definition.comodel,
+            };
+            const field = getFieldFromRegistry(definition.type, widget);
+            let { fieldsToFetch } = field;
+            if (fieldsToFetch) {
+                if (fieldsToFetch instanceof Function) {
+                    fieldsToFetch = fieldsToFetch({ options: {}, attrs: {} });
+                }
+                fieldsToFetch = Object.fromEntries(fieldsToFetch.map((f) => [f.name, f]));
+            }
+            this.activeFields[propertyFieldName] = {
+                ...pick(
+                    this.activeFields[fieldName],
+                    "context",
+                    "viewType",
+                    "help",
+                    "modifiers",
+                    "onChange",
+                    "forceSave",
+                    "decorations",
+                    "noLabel",
+                    "props",
+                    "alwaysInvisible"
+                ),
+                name: propertyFieldName,
+                string: definition.string,
+                relatedPropertyField,
+                attrs: {},
+                options: {},
+                propertyName: definition.name,
+                widget,
+                fieldsToFetch,
+                field,
+                tags: definition.tags,
```

Comment:
would prefer not to see this here (it's not a "node information"), same for relatedPropertyField maybe